### PR TITLE
[BACK-464] Add Sentry integration

### PR DESCRIPTION
**Pull request by mmiermans**
## Goal
Integrate Sentry to give better insight into errors and exceptions.

## All Submissions:

- [x] Have you followed the guidelines in our Contributing document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?

### CODE REVIEW

**Review by bassrock — CHANGES_REQUESTED** [review]
@mmiermans Can we pair on this? There's actually a much much easier approach you can take with the ECS task definition for using secrets.

Instead, there is an option called Secret Environment Variables that you can pass in the task definition and ECS will perform the secret decryption for you and inject the value as an environment variable only the task can see the value of.

https://aws.amazon.com/premiumsupport/knowledge-center/ecs-data-security-container-task/

**Review by kkyeboah — COMMENTED** [review]
Based on @bassrock's comment, there appears to be a simpler way to get secrets in ECS containers. Commenting to remove from queue.

**Review by jpetto — COMMENTED** [review]
commenting to remove from review queue. feel free to re-ping after addressing daniel's comment.

**Review by bassrock — DISMISSED** [review]
👍 paired together. Looks great to me!

File: cloudformation/proxy-service.yaml [hunk 1]
```diff
@@ -296,11 +315,6 @@ Resources:
                 Resource:
                   - !Sub "arn:aws:s3:::${EcsAppGeoipS3Bucket}"
                   - !Sub "arn:aws:s3:::${EcsAppGeoipS3Bucket}/*"
-              - Effect: Allow
```
Comment (bassrock) [inline cloudformation/proxy-service.yaml:299]: @mmiermans lets leave this in. If we remove it then we need to time deploys perfectly, but if we leave it we can deploy the code change at will.

**Review by bassrock — COMMENTED** [review]
Just had one comment.

**Review by bassrock — APPROVED** [review]

**Pull request merged**